File: homework_2/sumOfDigits.py
def sumOfDigits(number):
    remainder = 0
    result = 0
    
    while number > 0:
        remainder = number % 10
        number //= 10
        
        result += remainder
        
    return result

def main():
    number = int(input("Enter a number: "))
    
    result = sumOfDigits(number)
    
    print("The sum of the digits is", result)
    
    return 0
    
if __name__ == "__main__":
    main()    

File: homework_2/sumOfDigits.c
#include <stdio.h>

int sum(int initial) {
    int remainder;
    int result = 0;

    while(initial > 0) {
        remainder = initial % 10; //assing the last digit of the initial number to remainder
        initial = initial / 10; //remove the last digit of initial

        result += remainder; //add the remainder to the result
    }
    return result;
}

int main() {
    int number;

    printf("Enter an integer: ");
    scanf("%d", &number);

    int result = sum(number);

    printf("The sum of digits is %d \n", result);
    return 0;
}

